app/helpers/route_overrides_helper.rb

```diff
@@ -0,0 +1,37 @@
+module RouteOverridesHelper
+  class << self
+    private
+
+    def map_path(bad, *args)
```

Comment:
Assignment Branch Condition size for map_path is too high. [28.6/15]<br>Method has too many lines. [20/10]